src/notification.c

```diff
@@ -16,7 +16,7 @@
 
 #define COLUMN_COUNT 8
 
-#define X_CANONICAL_PRIVATE_SYNCHRONOUS "x-canonical-private-synchronous"
+#define X_CANONICAL_PRIVATE_SYNCHRONOUS "x-ayatana-private-synchronous"
```

Comment:
this one should not be changed. As there are not x-ayatana/canonical-types used in a-i-notifications, I will close this PR directly unmerged.